coralproject/talk
# Flags: Implement Labels, bugfixes and improvements

## What does this PR do?
- New slot `adminCommentLabels`
- Fix toxic comments not adding a flag and not appearing in reported queue
- Fix `all` comments queue and `userDetail` `all` tab not showing all comments.
- Fix mod queue `loadMore` feature not considering tags.
- Return actions for `CreateComment` Mutation
- Posting a toxic comment twice will show a different notification now and not the default premod notification
- Show toxic comment flag label
- Implement core labels:
  - reply label
  - premod label
  - history flag label
  - suspect flag label
  - user flag label

## How do I test this PR?

- Post a toxic comment (twice), see new notification text that states that your text might still violates guidelines etc.
- Check the existance of the above mentioned labels in the modqueue and user detail
- Check that the `all` moderation tab contains  `SYSTEM_WITHHELD` comments
- Test as a priviledged and as a normal user.
